docs/querying/sql-json-functions.md

```diff
@@ -47,7 +47,7 @@ You can use the following JSON functions to extract, transform, and create `COMP
 
 ### JSONPath syntax
 
-Druid supports a subset of the [JSONPath syntax](https://github.com/json-path/JsonPath/blob/master/README.md) operators, primarily limited to extracting individual values from nested data structures.
+Druid supports a subset of the [JSONPath syntax](https://github.com/json-path/JsonPath/blob/master/README.md) operators, primarily limited to extracting individual values from nested data structures. You can't use wildcards, invoke functions, or filter expressions.
```

Comment:
hmm, personally I think this seems a redundant since the table immediately after says exactly what we do support.

I wonder if we should clarify that the subset of jsonpath that we support here is totally distinct from what expressions are supported via `flattenSpec`, but this doesn't really seem like the most approprate place to discuss that since is a native ingestion concept.

Reply:
We got a specific query about it so I thought it was worth clarification.